src/features/suggestProduct/SuggestProduct.tsx

```diff
@@ -0,0 +1,126 @@
+import {useFormik} from 'formik';
+import * as Yup from 'yup';
+import {useQuery} from "@tanstack/react-query";
+import {getAllCategories} from "../../services/api.ts";
+import {useNavigate} from "react-router-dom";
+
+interface ProductSuggestion {
+    title: string;
+    price: number;
+    description: string;
+    category: string;
+}
+
+const SuggestProductForm = () => {
+    const categoriesQuery = useQuery<string[]>({
+        queryKey: ['categories'],
+        queryFn: getAllCategories
+    });
+
+    const navigate = useNavigate();
+
+    const formik = useFormik<ProductSuggestion>({
+        initialValues: {
+            title: '',
+            price: 0.00,
+            description: '',
+            category: ''
+        },
+        validationSchema: Yup.object({
+            title: Yup.string().required('Title is required'),
+            price: Yup.number().required('Price is required').positive('Price must be positive'),
+            description: Yup.string().required('Description is required'),
+            category: Yup.string().required('Category is required')
+        }),
+        onSubmit: (values) => {
+            console.log('Form values:', values);
+            alert(`${values.title} Product suggested successfully!`);
```

Comment:
Alerts are fairly sucky user experiences because they force the user to stop what they're doing. If you can, rather use a toast or notification.